azure-pipelines-templates/deploy/step/app-reg-new-temp-client-secret.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
```

Comment:
Do you want to remove all the duff files we don't use?

![image](https://github.com/SkillsFundingAgency/das-payments-v2-monitoring-alerts/assets/2112234/a11f650f-50a4-4a44-8e42-60a7f67b8dd3)

Screenshot from EarningEvents on the files left after removing all the unused ones.

Reply:
yes, please remove